tasks/rh-sign-cosign-image/rh-sign-cosign-image.yaml

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rh-sign-cosign-image
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to sign container images in a Snapshot spec using the cosign.
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+    - name: secretName
+      description: Name of secret containing needed credentials 
+      type: string
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: sign-image
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+      env:
+        - name: AWS_REGION
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_REGION
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_ACCESS_KEY_ID
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_SECRET_ACCESS_KEY
+      script: |
+        #!/usr/bin/env bash
+        set -eu
+        function fetch_cosign_key() {
+          echo $(kubectl_get_arg_from_config_map $1 SIG_KEY)
```

Comment:
Not sure how this is supposed to work. `kubectl_get_arg_from_config_map` is undefined here (it's only defined in context of tests via mocks.sh).

Reply:
Right, I forgot to change this one

Reply:
fixed

Reply:
I added a new comment there.